src/ProxyClient/Invalidation/ClearCacheCapable.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\ProxyClient\Invalidation;
+
+use FOS\HttpCache\ProxyClient\ProxyClient;
+
+/**
+ * An HTTP cache that supports clearing all of its
+ * cache entries.
+ *
+ * Proxy clients supporting this interface can allow
+ * for a more efficient delete-all operation rather
+ * than banning everything. Also it serves as
+ * an alternative to proxies that do not support
+ * banning cache entries at all.
+ */
+interface ClearCacheCapable extends ProxyClient
```

Comment:
to be consistent with the other interfaces and our naming, how about `ClearCapable`?

Reply:
Right, it's all about the `cache` anyway here so that makes no sense. I've renamed it. However, not sure if I should also rename the method from `clearCache()` to `clear()`?

Reply:
lets rename the method on the proxy client to `clear` but keep the cache invalidator method `clearCache` i'd say.

Reply:
Done :)